### PR TITLE
Create devcontainer

### PR DESCRIPTION
**Pull request by jarrodcolburn**
Should be able to run `./dash_site` once the devcontainer is up.
Hopefully will encourage people to help with the site.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+		"PATH": "${containerEnv:PATH}:${containerEnv:FLUTTER_ROOT}/bin"
+	},
+	"onCreateCommand": "git clone --branch=stable https://github.com/flutter/flutter.git ${containerEnv:FLUTTER_ROOT} && ${containerEnv:FLUTTER_ROOT}/bin/flutter doctor",
+	"postCreateCommand": "git submodule update --init --recursive && flutter pub get && pnpm install",
+	"customizations": {
+		"vscode": {
+			"extensions": ["Dart-Code.dart-code","Dart-Code.flutter"]
+		}
+	}
+}
```
Comment (domesticmouse) [inline .devcontainer/devcontainer.json:22]: Nit: please add LF at EOF

**Review by domesticmouse — APPROVED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (jarrodcolburn) [reply .devcontainer/devcontainer.json]: 👍

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Pull request merged**